### PR TITLE
feat: add `toml_file` table

### PR DESCRIPTION
**Pull request by vil02**
This PR adds a new table: `toml_file`.

<details>
  <summary>Results</summary>

```
+---------------------------------------------------------------------------+
| file_content                                                              |
+---------------------------------------------------------------------------+
| {                                                                         |
|     "tool": {                                                             |
|         "poetry": {                                                       |
|             "name": "puzzle-generator",                                   |
|             "group": {                                                    |
|                 "dev": {                                                  |
|                     "optional": true,                                     |
|                     "dependencies": {                                     |
|                         "mypy": "1.11.2",                                 |
|                         "ruff": "0.6.9",                                  |
|                         "bandit": "1.7.10",                               |
|                         "flake8": "7.1.1",                                |
|                         "pylint": "3.3.1",                                |
|                         "pytest": "8.3.3",                                |
|                         "coverage": "7.6.1",                              |
|                         "flake8-bugbear": "24.8.19",                      |
|                         "flake8-pytest-style": "2.0.0"                    |
|                     }                                                     |
|                 }                                                         |
|             },                                                            |
|             "readme": "./puzzle_generator/README.md",                     |
|             "authors": [                                                  |
|                 "piotr.idzik <vil02_puzzle_generator@10g.pl>"             |
|             ],                                                            |
|             "license": "MIT",                                             |
|             "version": "0.12.4",                                          |
|             "keywords": [                                                 |
|                 "puzzle-generation",                                      |
|                 "quiz-generation",                                        |
|                 "code-generation"                                         |
|             ],                                                            |
|             "packages": [                                                 |
|                 {                                                         |
|                     "include": "puzzle_generator"                         |
|                 }                                                         |
|             ],                                                            |
|             "repository": "https://github.com/vil02/puzzle_generator",    |
|             "description": "Generates python code representing a puzzle", |
|             "dependencies": {                                             |
|                 "black": "24.8.0",                                        |
|                 "python": ">=3.10,<4.0.0"                                 |
|             }                                                             |
|         }                                                                 |
|     },                                                                    |
|     "build-system": {                                                     |
|         "requires": [                                                     |
|             "poetry-core"                                                 |
|         ],                                                                |
|         "build-backend": "poetry.core.masonry.api"                        |
|     }                                                                     |
| }                                                                         |
+---------------------------------------------------------------------------+
```
query:
```sql
select jsonb_pretty(content) as file_content from toml_file where path='pyproject.toml'
```
(cf. [`pyproject.toml`](https://github.com/vil02/puzzle_generator/blob/master/pyproject.toml)).

</details>

### CODE REVIEW

**Comment by ParthaI** [comment]
Hello @vil02, thank you so much for your contribution 🎉.

The overall changes look great to me. Could you please include an example of the query result in the PR description?

**Comment by vil02** [comment]
@ParthaI: I updated the PR description. I though that the example is redundant, since there are some in the documentation.

File: docs/index.md [hunk 1]
```diff
@@ -270,6 +323,7 @@ connection "config" {
 
   ini_paths  = [ "*.ini", "~/*.ini", "/path/to/dir/main.ini" ]
   json_paths = [ "*.json", "~/*.json", "/path/to/dir/main.json" ]
+  toml_paths = [ "*.toml", "/path/to/dir/main.toml" ]
```
Comment (ParthaI) [inline docs/index.md:326]:
```suggestion
  toml_paths = [ "*.toml", "~/*.toml", "/path/to/dir/main.toml" ]
```

Could we consider the above suggestions to maintain consistency?

**Review by ParthaI — COMMENTED** [review]
Hi @vil02, I've left a minor review comment. Please take a look. Thanks!

File: docs/index.md (hunk 1 above)
Comment (vil02) [reply docs/index.md:326]: > Could we consider the above suggestions to maintain consistency?

Sure, thanks for spotting that!

**Review by ParthaI — APPROVED** [review]

**Review by ParthaI — APPROVED** [review]

**Comment by vil02** [comment]
@ParthaI: please let me know if there is still something, which has to be done here.

**Comment by ParthaI** [comment]
Apologies for the delay, @vil02... @misraved, could you please review this PR and proceed if everything looks good to you? Thanks!

**Pull request merged**